Client/DropboxSdkClient.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Dizda\CloudBackupBundle\Client;
+
+use Symfony\Component\Console\Output\ConsoleOutput;
+use \Dropbox as dbx;
+
+/**
+ * Class DropboxClient.
+ *
+ * @author  Jonathan Dizdarevic <dizda@dizda.fr>
+ */
+class DropboxSdkClient implements ClientInterface
+{
+    private $access_token;
+
+    /**
+     * @param array $params user
+     */
+    public function __construct($params)
+    {
+        $params             = $params['dropbox_sdk'];
+        $this->access_token = $params['access_token'];
+        $this->remotePath   = $params['remote_path'];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function upload($archive)
+    {
+        $fileName = explode('/', $archive);
+        
+        $pathError = dbx\Path::findErrorNonRoot($this->remotePath);
+        if ($pathError !== null) {
+            fwrite(STDERR, "Invalid <dropbox-path>: $pathError\n");
+            die;
```

Comment:
You better use Exceptions, see https://github.com/dizda/CloudBackupBundle/blob/master/Client/GoogleDriveClient.php#L132 rather than L35-36.